.prettierignore

```diff
@@ -1,10 +1,13 @@
-# Ignore a bunch of files:
-*.html
-*.json
-*.handlebars
-*.css
-.bablerc
-**/CHANGELOG.md
+# Ignore everything by default
+**
+
+# Include only specific extensions
+!*.{md,mdx,js,jsx,ts,tsx,json,css,html}
```

Comment:
I see that we're technically adding html, css and json by making this change, but since prettier is passing, I can't see any reason to keep them ignored.